From 4e876b3561f1485d9216d436157af110c910c764 Mon Sep 17 00:00:00 2001
From: Igor Grinberg <igrinberg@habana.ai>
Date: Wed, 01 Mar 2023 17:09:23 +0200
Subject: [PATCH] [SW-99614] erom_recovery: remove redundant call

The preboot_security_sp_sram_init() is already called from
preboot_security_init(), there is no need to call it again.
Remove the redundant call.

Change-Id: I0aa65d8714191e3e2442dd806b50b0b866a71213
Signed-off-by: Igor Grinberg <igrinberg@habana.ai>
---

diff --git a/app/ext_rom/erom_recovery.c b/app/ext_rom/erom_recovery.c
index 8c2f783..f71ae57 100644
--- a/app/ext_rom/erom_recovery.c
+++ b/app/ext_rom/erom_recovery.c
@@ -125,8 +125,6 @@
 	/* Prepare boot FW versions and eFuse data for host */
 	preboot_version();
 
-	preboot_security_sp_sram_init();
-
 	preboot_clear_wd_gpio();
 
 	/* In Preboot, module LKD is responsible for setting the below flag */
